discojs/discojs-core/src/informant/training_informant/base.ts

```diff
@@ -71,6 +73,17 @@ export abstract class Base {
     return this.validationGraphInformant.accuracy()
   }
 
+  /** add loss for current round */
+  // eslint-disable-next-line accessor-pairs
+  set loss (loss: number) {
+    this._losses = this._losses.push(loss)
```

Comment:
I think this syntax is quite misleading and leads to inconsistent getter - setter:
```js
trainingInformant.loss = logs.val_loss // actually pushes val_loss in a list  rather than set the value
console.log(trainingInformant.loss) // prints undefined
console.log(trainingInformant.losses.last() // this is the right syntax the value I just assigned, which I find inconsistent
```
This snippet explains why an undefined loss is being printed during training.
What do you think?